packages/dds/tree/src/core/tree/deltaUtil.ts

```diff
@@ -3,6 +3,7 @@
  * Licensed under the MIT License.
  */
 
+import type { FieldChangeDelta } from "../../feature-libraries/index.js";
```

Comment:
modules in /feature-libraries are supposed to depend on /core, but not the other way around.
Move emptyFieldChanges to src\feature-libraries\modular-schema\fieldChangeHandler.ts

Reply:
I ended up just moving emptyFieldChanges to the one test file that uses it